Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.36928&attempt_id=9c9f24ca-a33f-4704-8e53-d6973e57cbe9)

Attempted: pcm

<details><summary>Partial log (click to expand)</summary><p>

```
In file included from cpucounters.cpp:32:0:
cpucounters.h: In member function 'uint64 PCM::getTickCountRDTSCP(uint64)':
cpucounters.h:1542:62: error: unknown register name '%rdx' in 'asm'
            "=r" (high), "=r" (low) :: "%rax", "%rcx", "%rdx");
                                                              ^
cpucounters.h:1542:62: error: unknown register name '%rcx' in 'asm'
cpucounters.h:1542:62: error: unknown register name '%rax' in 'asm'
make: *** [Makefile:56: cpucounters.o] Error 1
builder for '/nix/store/kb9yb4r4xpkbr6xdzps15xiq8q49nj5a-pcm-201710.drv' failed with exit code 2
[31;1merror:[0m build of '/nix/store/kb9yb4r4xpkbr6xdzps15xiq8q49nj5a-pcm-201710.drv' failed
```
</p></details>

--- a/pkgs/os-specific/linux/pcm/default.nix
+++ b/pkgs/os-specific/linux/pcm/default.nix
@@ -1,0 +1,26 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  version = "201710";
+  name = "pcm-${version}";
+
+  src = fetchFromGitHub {
+    owner = "opcm";
+    repo = "pcm";
+    rev = "${version}";
+    sha256 = "02rq8739zwwbfrhagvcgf6qpmnswxl9b0qsld26rg6zp91v2npbj";
+  };
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp pcm*.x $out/bin
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Processor counter monitor";
+    homepage = https://www.intel.com/software/pcm;
+    license = licenses.bsd3;
+    maintainers = with maintainers; [ roosemberth ];
+    platforms = [ "x86_64-linux" ];
+  };
+}

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -13092,6 +13092,8 @@ with pkgs;
 
   i7z = callPackage ../os-specific/linux/i7z { };
 
+  pcm = callPackage ../os-specific/linux/pcm { };
+
   ima-evm-utils = callPackage ../os-specific/linux/ima-evm-utils { };
 
   intel2200BGFirmware = callPackage ../os-specific/linux/firmware/intel2200BGFirmware { };

